google-cloud-compute/src/test/java/com/google/cloud/compute/it/ITComputeTest.java

```diff
@@ -1155,9 +1193,13 @@ public void testListDisksAndSnapshots() throws InterruptedException, TimeoutExce
     }
     assertEquals(2, count);
     compute.deleteDisk(firstDiskId);
+    managedResourceCleaner.remove(firstDiskId);
     compute.deleteDisk(secondDiskId);
+    managedResourceCleaner.remove(secondDiskId);
     compute.deleteSnapshot(firstSnapshotId);
+    managedResourceCleaner.remove(firstSnapshotId);
     compute.deleteSnapshot(secondSnapshotId);
+    managedResourceCleaner.remove(secondSnapshotId);
```

Comment:
For tests where there is no check after deletion, you could just remove the deletes altogether, since the resource cleaner will take care of it in `tearDown`.